wren-ai-service/README.md

```diff
@@ -42,9 +42,19 @@ Please read the [documentation](https://docs.getwren.ai/concept/wren_ai_service)
   - restart wren ai service
   - check `outputs` folder for the evaluation result file
 - to run the load test
-  - in the demo folder, run `make prepare` to start the docker containers
-  - in the demo folder, run `make ai` to start the ai service
+  - Choose between the 2 setup options:
+      - Setup option 1: demo env setup
+        - in the demo folder, run `make prepare` to start the docker containers
+        - in the demo folder, run `make ai` to start the ai service
```

Comment:
I suggest we only provide the second setup, since we don't provide the way to ingest dataset for the first setup.

Reply:
Sure, let me adjust it.